pkg/util/containers/providers/cgroup/cgroup_metrics.go

```diff
@@ -24,6 +26,10 @@ import (
 // TODO: get USER_HZ from gopsutil? Needs to patch it
 const NanoToUserHZDivisor float64 = 1e9 / 100
 
+var (
+	numCPU float64 = float64(runtime.NumCPU())
```

Comment:
this value is the number of cpu seen by the agent?

In the context of running in a container? does the value may be different?

Reply:
Yes it's the value seen by the agent. In my knowledge there's no way to hide/modify it in a container (unlike VMs - it's probably coming from `/proc/cpuinfo`), so it's always number of CPUs on the host.